storybook/stubs/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -1,3 +1,5 @@
 import QtQuick 2.15
 
-QtObject {}
+QtObject {
+    property var chatCommunitySectionModule
```

Comment:
Is it really needed here?

Reply:
Yes, it doesn't run in storybook without it